nonCisMaleInCs/src/Navigation/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
why `package.json` inside `/Navigation` folder?

Reply:
Instead of declaring the navigation function on each component, I declared all of our functions in Navigation/index.js, and to easily import this file added the `package.json` file to the Navigation folder.